website/docs/r/serverless_instance.html.markdown

```diff
@@ -24,6 +24,7 @@ resource "mongodbatlas_serverless_instance" "test" {
   provider_settings_backing_provider_name = "AWS"
   provider_settings_provider_name = "SERVERLESS"
   provider_settings_region_name = "US_EAST_1"
+  continuous_backup_enabled = true
```

Comment:
We have included a lot of optional parameters in examples and later regretted it.  So let's not add them unless it's required.

Reply:
Removed continuous_backup_enabled